tck/pom.xml

```diff
@@ -980,4 +1047,11 @@
             </build>
         </profile>
     </profiles>
+    
+    <repositories>
```

Comment:
I think this can be left out. The staging repo is already defined (in a profile) in the parent pom. You can activate it on the command line using -Pstaging